show/main.py

```diff
@@ -1383,8 +1391,20 @@ def runningconfiguration():
 @click.option('--verbose', is_flag=True, help="Enable verbose output")
 def all(verbose):
     """Show full running configuration"""
-    cmd = "sonic-cfggen -d --print-data"
-    run_command(cmd, display_cmd=verbose)
+    cmd = ['sonic-cfggen', '-d', '--print-data']
+    stdout = get_cmd_output(cmd)
```

Comment:
>get_cmd_output [](http://example.com/codeflow?start=13&length=14)

Original behavior is that even rc is non-zero, the command output is printed as output. Do you  want to keep the behavior?